src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Bit.Client.Web.BlazorUI
+{
+    public partial class BitNav
+    {
+        [Parameter] public string SelectedKey { get; set; }
+        [Parameter] public string AriaLabel { get; set; }
+        [Parameter] public bool IsOnTop { get; set; }
+        [Parameter] public List<NavLink> NavLinks { get; set; }
+        [Parameter] public EventCallback<NavLink> OnClick { get; set; }
+        [Parameter] public Func<NavLink, RenderFragment> OnRenderHeader { get; set; }
```

Comment:
This property should be defined as a RenderFragment<NavLink> and rename to "HeaderTemplate".
there is no need to define another method to call it, just use "@HeaderTemplate(navLink)" in the .razor file.
at the consumer side (in Component.razor) the property can be assigned simply like this:

```cs
    <BitNav NavLinks="CustomHeaderLinks"
            AriaLabel="Nav with custom group header">
        <HeaderTemplate Context="link">
            <h3> @link.Name </h3>
        </HeaderTemplate>
    </BitNav>
```